File: app/src/main/java/mariavv/fitnesspal/ui/journal/daycard/ItemType.java
package mariavv.fitnesspal.ui.journal.daycard;

import android.support.v7.widget.RecyclerView;

public interface ItemType {
    int HEADER = 0;
    int MEAL = 1;
    int DISH = 2;

    int getItemViewType();

    void onBindViewHolder(RecyclerView.ViewHolder viewHolder);
}

File: app/src/main/java/mariavv/fitnesspal/presentation/journal/daycard/ItemType.kt
package mariavv.fitnesspal.presentation.journal.daycard

import android.support.v7.widget.RecyclerView

interface ItemType {

    val itemViewType: Int

    fun onBindViewHolder(viewHolder: RecyclerView.ViewHolder)

    companion object {
        val HEADER = 0
        val MEAL = 1
        val DISH = 2
    }
}
